python/tank/bootstrap/cached_configuration.py

```diff
@@ -278,6 +279,13 @@ def update_configuration(self):
             # make sure the config is locally available.
             self._descriptor.ensure_local()
 
+            # Now cache the core as well. The goal
+            DescriptorOperations(
```

Comment:
i feel we should figure out a better name for this. But i don't have any great suggestions right now..